### PR TITLE
Feature/18/mock data

### PR DESCRIPTION
**Pull request by pilahr**
Hi, we created mock data for staffs and clients. Please review.

### CODE REVIEW

**Review by olirob93 — CHANGES_REQUESTED** [review]
Can you put all the data into one file called mockData which would be one object with arrays of objects. I can show you this if you wish.

Also, can you get images for each of the staff members, ChatGPT should be able to do this for you using Unsplash.

File: src/assets/data/mockData.js [hunk 1]
```diff
@@ -0,0 +1,323 @@
+import staffImage from "../images/staff-img.svg";
```
Comment (olirob93) [inline src/assets/data/mockData.js]: This is no longer needed

**Review by olirob93 — COMMENTED** [review]
If you change that final thing. I will approve :)

File: src/assets/data/mockData.js (hunk 1 above)
Comment (Alan-day) [reply src/assets/data/mockData.js]: done

**Review by olirob93 — APPROVED** [review]

**Pull request merged**